Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -179,6 +180,18 @@ final class StatusTableViewController: LoopChartsTableViewController {
         
         deviceManager.checkDeliveryUncertaintyState()
     }
+    
+    private func authorizeHealthStore(completion: @escaping (Bool) -> Void) {
+        deviceManager.authorizeHealthStore { accessFormWasCompleted in
+            // returned Bool only indicates if the user completed the health access form.
+            if accessFormWasCompleted {
+                completion(accessFormWasCompleted)
+            } else {
+                // if the user did not complete the health access form, present the health access form again so the user can allow or deny access
+                self.authorizeHealthStore(completion: completion)
```

Comment:
How would this happen? How could the form be displayed and dismissed, but not allowed or denied access?

Reply:
There are steps to reproduce in the ticket, which include pressing the home button.